langchain-core/src/runnables/base.ts

```diff
@@ -521,7 +521,7 @@ export abstract class Runnable<
         async () =>
           callbackManager_?.handleChainStart(
             this.toJSON(),
-            { input: "" },
+            finalInput,
```

Comment:
There are cases where `finalInput` is undefined right?

Should this fall back to the old value?